db/tests/types/operations/test_cast.py

```diff
@@ -447,6 +465,48 @@
             },
         },
     },
+    TIMESTAMP_WITHOUT_TIME_ZONE: {
+        ISCHEMA_NAME: PostgresType.TIMESTAMP_WITHOUT_TIME_ZONE.value,
+        REFLECTED_NAME: TIMESTAMP_WITHOUT_TIME_ZONE,
+        TARGET_DICT: {
+            CHAR: {VALID: []},
```

Comment:
I presume a timestamp should be castable to any text/character type, including `CHAR`.

Reply:
It is possible to cast a `timestamp` to `CHAR`, but `CHAR` returns exact number of characters specified and this causes problems with testing as the `type_options` has to be specified with each test case, which is not possible right now. I have created a [issue](https://github.com/centerofci/mathesar/issues/869) that should help with testing such cases